### PR TITLE
Use OpenSSL to un/protect srtp packets and upgrade libsrtp to v2.1

### PR DESCRIPTION
**Pull request by jcague**
**Description**

Upgrade libsrtp to 2.1 version. And use OpenSSL to protect and unprotect srtp packets.

[] It needs and includes Unit Tests

**Changes in Client or Server public APIs**

Not needed

[] It includes documentation for these changes in `/doc`.

### CODE REVIEW

File: scripts/installMacDeps.sh [hunk 1]
```diff
@@ -178,7 +188,7 @@ install_mediadeps(){
 }
 
 install_mediadeps_nogpl(){
-  brew install opus libvpx
+  # brew install opus libvpx
```
Comment (lodoyun) [inline scripts/installMacDeps.sh]: Is this a typo?

File: scripts/installMacDeps.sh [hunk 2]
```diff
@@ -205,7 +215,7 @@ pause "Installing homebrew..."
 install_homebrew
 
 pause "Installing deps via homebrew..."
-install_brew_deps
+# install_brew_deps
```
Comment (lodoyun) [inline scripts/installMacDeps.sh]: Again, is there a reason to change this?

**Comment by kekkokk** [comment]
```
make[2]: *** No rule to make target `/home/ubuntu/licode/build/libdeps/build/lib/libsrtp.a', needed by `erizo/liberizo.so'.  Stop.
make[1]: *** [erizo/CMakeFiles/erizo.dir/all] Error 2
make: *** [all] Error 2

```

**Comment by lodoyun** [comment]
You have to reinstall the dependencies (`installUbuntuDeps`).
Make sure you clean up the erizo build by removing `licode/erizo/build`
We are no longer including libsrtp in the repository and is now downloaded as part of the installation scripts

**Comment by kekkokk** [comment]
`Make sure you clean up the erizo build by removing licode/erizo/build`

that did the trick, thanks

**Pull request merged**